pkg/ccl/crosscluster/logical/dead_letter_queue.go

```diff
@@ -103,6 +104,7 @@ func (dlq *loggingDeadLetterQueueClient) Log(
 	ingestionJobID int64,
 	kv streampb.StreamEvent_KV,
 	cdcEventRow cdcevent.Row,
+	reason error,
 	dlqReason retryEligibility,
```

Comment:
nit: rename this var to `stoppedRetryReason` so the var name is consistent across implementations